webapp/components/interop-dashboard.js

```diff
@@ -244,6 +244,10 @@ class InteropDashboard extends WPTFlags(PolymerElement) {
           border-radius: 5px;
         }
 
+        #mobileWarning {
+          margin-bottom: 32px;
+        }
```

Comment:
Perf/simplicity nit: since you are adding this style, better add the .warning bits here as well, since the latter is now redundant.